### DISCUSSION TITLE
Does this send any data outside of localhost?

### DISCUSSION THREAD

**Discussion by lamemantis** [hacksider/Deep-Live-Cam#340] [Q&A]
Can it run without internet? What data does it send and where?

Id rather not have my photos be sent out to another server just to test this out. So if someone can clarify this id appreciate it

**Comment by hacksider** [comment]
You can monitor your network as well as check the code ☺️ that's the reason this is open sourced.

Anyway, to answer your question. It doesn't need to send data outside the network.

I also suggest blocking outgoing data if you have any doubts.
